scriptmodules/supplementary/raspbiantools.sh

```diff
@@ -73,6 +76,19 @@ function stretch_fix_raspbiantools() {
     fi
 }
 
+function buster_bluez_pin_raspbiantools() {
+    # pin the 'bluez' package to the RPI repos to prevent any Debian updates overwriting it
+    # use Priority 1001 to force the the installation even when the Debian package is installed
+    local pin_file="/etc/apt/preferences.d/01-bluez-pin"
+    if isPlatform "rpi" && [[ "$__os_debian_ver" -eq 10 ]] && ! -f "$pin_file" ; then
```

Comment:
Should this be `    if isPlatform "rpi" && [[ "$__os_debian_ver" -eq 10 && ! -f "$pin_file" ]] ; then` ?

Reply:
Yes, I corrected it.